### DISCUSSION TITLE
ESP32-A1S operation from battery on BAT connector

### DISCUSSION THREAD

**Discussion by Androssboy** [sle118/squeezelite-esp32#441] [Q&A]
Hello, my friends!
First of all, I want to thank all the enthusiasts for their efforts!
Colleagues, together with you I have come a long way and overcome many obstacles, but one of the problems has driven me into a dead end. In my project, I chose the Espressif ESP32 A1S Audio devKit v2.2. board as a basis and am already at the end of the design and prototyping stage. Even the case is ready (WOW!!!). So, when switching the board's power supply to the battery, which is connected to the standard BAT connector, the encoder and IR remote control stop working. Does anyone have any ideas on how to fix this?

**Comment by philippe44** [comment]
That board has many version, even for the same release. There is probably a GPIO/ADC conflict here.

**Reply by sle118** [reply]
Agreed here. I have several boards based on the A1S and there seems to be slight variations between each one.
